securetar/__init__.py

```diff
@@ -276,7 +281,7 @@ def __exit__(self, exc_type, exc_value, traceback) -> None:
 
 @contextmanager
 def _add_stream(
-    tar: tarfile.TarFile, tar_info: tarfile.TarInfo
+    tar: tarfile.TarFile, tar_info: tarfile.TarInfo, padding: bytearray
```

Comment:
I am a bit confused why we add the pax header depending on this padding parameter. What is it used for? Maybe we can extend the docs of this function as well?

Reply:
OK, I will try to make this clearer.

Reply:
If the inner tar file is encrypted, we add the size of the plaintext as a custom header.

Instead of giving the outer tar file streamer access to the padding data, we could pass in some mutable container which has:
- A flag indicating if the inner tar is encrypted
- A flag with the size of the padding

This might be easier to understand?